modules/imap/output_modules.php

```diff
@@ -239,12 +239,7 @@ protected function output() {
                 '<a class="hlink small_headers" style="display: none;" href="#">'.$this->trans('Small headers').'</a>';
             if (!isset($headers['Flags']) || !stristr($headers['Flags'], 'draft')) {
                 $txt .= ' | <a class="reply_link hlink" href="?page=compose&amp;reply=1'.$reply_args.'">'.$this->trans('Reply').'</a>';
-                if ($reply_all || $size > 1) {
-                    $txt .= ' | <a class="reply_all_link hlink" href="?page=compose&amp;reply_all=1'.$reply_args.'">'.$this->trans('Reply-all').'</a>';
-                }
-                else {
-                    $txt .= ' | <a class="reply_all_link hlink disabled_link">'.$this->trans('Reply-all').'</a>';
-                }
+                $txt .= ' | <a class="reply_all_link hlink" href="?page=compose&amp;reply_all=1'.$reply_args.'">'.$this->trans('Reply-all').'</a>';
```

Comment:
We still need to check if there is more than one sender to reply

```php
if ($reply_all || $size > 1) {
   $txt .= ' | <a class="reply_all_link hlink" href="?page=compose&amp;reply_all=1'.$reply_args.'">'.$this->trans('Reply-all').'</a>';
}
```

Reply:
If we still check without `else`, the **reply all** button will disappear when there is less than one message. 

It should be like that ?

Reply:
You're right, the previous pattern was to show so let's keep it that way. Thanks Jean

Reply:
You're right, the previous behaviour was to show it anyway so let's keep it that way. Thanks Jean

Reply:
Thank you too Henrique